### PR TITLE
Membership landing

### PR DESCRIPTION
**Pull request by malakali542**
Added membership cards -- still need content for cards

### CODE REVIEW

File: src/Components/LandingPage/Membership.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        <Grid
+          container
+          spacing={8}
+          alignItems="center"
+          justify="center"
+          className={classes.grid}
+          > 
+          <Grid
+            container
+            item xs={6} sm={6} md={6} lg={4}
```
Comment (ahmedhamodi) [inline src/Components/LandingPage/Membership.js:46]: Have you tested this on mobile screens? I feel like we would want xs={12} instead, but if it looks good in your opinion at 6, then it's fine.

File: src/Components/LandingPage/Membership.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            justify="center"
+          >
+            <MembershipCard
+              front_text="Mentee"
+              inner_text="Pricing Plan:"
+            />
+          </Grid>
+          <Grid
+            container
+            item xs={6} sm={6} md={6} lg={4}
```
Comment (ahmedhamodi) [inline src/Components/LandingPage/Membership.js:58]: Same here, xs 12 vs 6. Use your own discretion, I just thought I would point it out to ensure you checked it.

File: src/Components/LandingPage/MembershipCard.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+function withMyHook(Component){
+    return function WrappedComponent(props){
+        const classes = useStyles();
+        return <Component {...props} classes={classes}/>
+    }
+}
+
+class MembershipCard extends Component{
+  constructor(props) {
```
Comment (ahmedhamodi) [inline src/Components/LandingPage/MembershipCard.js:60]: If the constructor is just going to super it's props, it doesn't need to exist. That is the default behaviour. You can remove the constructor.

Comment (mohaimenhasan) [reply src/Components/LandingPage/MembershipCard.js:60]: Same here. Would get rid of the constructor block completely from this

File: src/Components/LandingPage/Membership.js (hunk 1 above)
Comment (malakali542) [reply src/Components/LandingPage/Membership.js:46]: Yeah just checked again- changing it to 12

**Comment by malakali542** [comment]
closes #26

**Pull request merged**